### PR TITLE
Update security.txt to a detailed SECURITY.md.

### PR DESCRIPTION
**Pull request by DailyDreaming**
See conversation here: https://github.com/HumanCellAtlas/dcp/issues/274

Other projects would link to this `SECURITY.md` file.

### CODE REVIEW

**Comment by DailyDreaming** [comment]
@kislyuk @adrazhi @sampierson Could I get someone to take a look at this?

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+This document provides guidelines on vulnerabilities in the components
+that the Human Cell Atlas project manages and how to report them.
+
+## Vulnerabilities
+
+Many of the Human Cell Atlas repos are a part of a large and complex system
+built on cloud architecture in AWS and GCP, whose vulnerabilities these projects
+are also subject to.  They also depend on a large set of third party libraries
+(e.g., `google-auth`, `Jinja2`, and others such as
+[this](https://github.com/HumanCellAtlas/data-store/blob/master/requirements.txt) ),
```
Comment (kislyuk) [inline SECURITY.md]: "such as this" -> "listed here"

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+Given the Human Cell Atlas's many components and projects, a vulnerability
+could occur in multiple places, and particularly in the parts making up
+our deployment platform, such as:
+[ingest](https://github.com/HumanCellAtlas/upload-service), 
+the [data-store](https://github.com/humancellatlas/data-store) itself, 
+the [data-browser](https://github.com/HumanCellAtlas/data-browser), 
+the [data-portal](https://github.com/HumanCellAtlas/data-portal).
+or the [dcp-cli](https://github.com/humancellatlas/dcp-cli).
+
+Public read access to files in, for example, data-store is intended and does
```
Comment (kislyuk) [inline SECURITY.md]: "Public read access" -> "By default, public read access"

**Comment by DailyDreaming** [comment]
@kislyuk Addressed, thanks for taking a look!

@sampierson @adrazhi Any comments on this?

**Pull request merged**